pycytominer/annotate.py

```diff
@@ -78,7 +78,11 @@ def annotate(
     platemap = load_platemap(platemap, add_metadata_id_to_platemap)
 
     annotated = platemap.merge(
-        profiles, left_on=join_on[0], right_on=join_on[1], how="inner"
+        profiles,
+        left_on=join_on[0],
+        right_on=join_on[1],
+        how="inner",
+        suffixes=["_platemap", None],
```

Comment:
Because pycytominer explicitly relies on certain column prefixes for analysis logic (e.g. Metadata_, Cells_, Cytoplasm_ all have special meaning), I don't think this suggest is feasible in this scenario.